src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -2710,7 +2713,12 @@ func (config *Config) appendQMPSockets() {
 			}
 		}
 
-		config.qemuParams = append(config.qemuParams, "-qmp")
+		if q.IsHmp {
+			config.qemuParams = append(config.qemuParams, "-monitor")
+		} else {
+			config.qemuParams = append(config.qemuParams, "-qmp")
```

Comment:
Can both be `-qmp`?

Reply:
Prefer `monitor` for debug, for the monitor interface is more readable for developer